www/views/hardware.html

```diff
@@ -1250,6 +1250,39 @@ <h1 data-i18n="Devices">Devices</h1>
 				</tr>
 			</table>
 		</div>
+		<div id="divgoodweweb">
+                        <br>
+                        <table class="display" id="hardwareparamsgoodweweb" border="0" cellpadding="0" cellspacing="20">
+                                <tr>
+					<td align="right" style="width:110px"><label><span data-i18n="username">Goodwe-power Username</span>:</label></td>
+					<td><input type="text" id="username" style="width: 170px; padding: .2em;" class="text ui-widget-content ui-corner-all" /></td>
+                                </tr>
+                        </table>
+                </div>
+<div id="divwinddelen">
```

Comment:
Fishy, I don't know what happened here. Will you issue a pull request or shall I do that?